### PR TITLE
Modify for fixing static code scan issues in CIV on WW_08 2020

### PR DESCRIPTION
**Pull request by Kexiaox**

### CODE REVIEW

**Comment by dmitryermilov** [comment]
And fix build please

**Comment by dmitryermilov** [comment]
And fix code alignment

File: _studio/mfx_lib/encode_hw/vp9/include/*Backtrace* [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Debugger entered--Lisp error:(void-variable wq)
```
Comment (dmitryermilov) [inline _studio/mfx_lib/encode_hw/vp9/include/*Backtrace*]: What is it?

File: _studio/mfx_lib/encode_hw/vp9/include/*Backtrace*~ [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Debugger entered--Lisp error: (void-variable wq)
```
Comment (dmitryermilov) [inline _studio/mfx_lib/encode_hw/vp9/include/*Backtrace*~]: same

**Comment by Kexiaox** [comment]
> I still would like to stack which calls static analyzer complains

line 255 in "mfx_enc_common.cpp" file

**Comment by dmitryermilov** [comment]
> line 255 in "mfx_enc_common.cpp" file

this line contains another function
https://github.com/Intel-Media-SDK/MediaSDK/blob/c3b4182a028945369e4a45befaa510a6cd7b5819/_studio/mfx_lib/shared/src/mfx_enc_common.cpp#L255

Please give good example

**Comment by LukeZheZhu** [comment]
> I still would like to stack which calls static analyzer complains

Hi @dmitryermilov,
This issue is caused by assert(p). The assert(p) works only when (#ifdef NDEBUG). In other situation, it will not work.

**Comment by dmitryermilov** [comment]
> > I still would like to stack which calls static analyzer complains
> 
> Hi @dmitryermilov,
> This issue is caused by assert(p). The assert(p) works only when (#ifdef NDEBUG). In other situation, it will not work.

Still, please share example of call stack

**Comment by dmitryermilov** [comment]
This is a false positive issue. mfxExtBufferRefProxy::operator T&() can be used only for cases when the buffer we're looking is available m_extParam. For cases when we're not sure , another getter function should be used: https://github.com/Intel-Media-SDK/MediaSDK/blob/97de8773fa0f66e7a2d0f9fd9de878fb698a5fbd/_studio/mfx_lib/shared/src/mfx_enc_common.cpp#L255

So right now I don't see any violations in the code of such rules.

**Pull request closed without merging**